src/main/java/com/checkmarx/flow/controller/GitHubController.java

```diff
@@ -161,7 +161,7 @@ public String pingRequest(
 
             setExclusionProperties(cxProperties, controllerRequest);
             //build request object
-            String gitUrl = repository.getCloneUrl();
+            String gitUrl = pullRequest.getHead().getRepo().getCloneUrl();
             String token = properties.getToken();
```

Comment:
This change fails when there is no clone url in the repo.

the line should be replaced with:
``` Java
String gitUrl = Optional.ofNullable(pullRequest.getHead().getRepo())
                    .map(Repo::getCloneUrl)
                    .orElse(repository.getCloneUrl());
```